### ISSUE TITLE
Human-readable deadline date format

### ISSUE THREAD

**Issue by wolfbeast** [PoC-Consortium/scavenger#97]
It would be a great enhancement if the dealines in the console/logging could be made a human readable format instead of the raw second value.
e.g. 
`7198` -> `01:59:58`
`123812` -> `1d 10:23:32`
etc.

**Comment by JohnnyFFM** [comment]
rejected 😁

two reasons: 

1. there is no value added... your miner just works, if you win a block you win. no point to watch it.

2. Each coin has a unique function to convert dl to time. it's not just divide by 60 anymore like it used to be at the beginning of PoC.

**Comment by wolfbeast** [comment]
I disagree with (1) but I wasn't aware of (2); in that case it's indeed not useful.
